problem_builder/questionnaire.py

```diff
@@ -93,13 +92,8 @@ def student_view(self, context=None):
         context['hide_header'] = context.get('hide_header', False) or not self.show_title
 
         fragment = Fragment(loader.render_template(template_path, context))
-        # If we use local_resource_url(self, ...) the runtime may insert many identical copies
-        # of questionnaire.[css/js] into the DOM. So we use the mentoring block here if possible
-        block_with_resources = self.get_parent()
-        if not isinstance(block_with_resources, MentoringBlock):
-            block_with_resources = self
```

Comment:
We normally work around circular dependencies by deferring imports until they are needed, for example: https://github.com/open-craft/problem-builder/blob/4ffa21c36b534f9b86f50b128815800a33b69899/problem_builder/mentoring.py#L418

Would that work here? If it works, it would be great if we could apply the same optimization to Step Builder.